unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
> Then what should be done about this?

This page should answer many questions:
https://github.com/micropython/micropython/wiki/ContributorGuidelines (link to it is shown on creation of pull request).